### DISCUSSION TITLE
Interesting, but No.

### DISCUSSION THREAD

**Discussion by mcbaer** [AppFlowy-IO/AppFlowy#417] [General]
It will be interesting to see where this project goes, but for now its a hard pass (no.)

Immediately after launching the "app" if felt like a poorly designed web-view instead of an actual app. Maybe this is just my personal preference of having things look like, which is why I use iOS/MacOS/iPadOS as my daily drivers for everything. 

Really should have left the "app" part off of the name. The icon also looks more like something you would see for a art related webapp (for like photo editing or something) as it looks like a painters palette.

Am I missing it, or does "flowy" lack the basic function of setting body (default) font size? I see a button for H1, H2, H3, but no option to change the font size of the body text. The current size is so large, like its made for semi-blind people. In comparison to the font size of the text Im typing current, the text size in "flowy" is at least several sizes larger.

Why is there no ability to just type in Markdown? I see an option to export a page to Markdown though. There is no option to right click text to set its style/block. Forced to move the hands off the keyboard to the mouse/trackpad to move the cursor all the way to the bottom of the window to click on one of the options in the toolbar. There should be a smooth flow to when a user is typing something, either keyboard shortcuts or just allow markdown support. Markdown would be one thing "flowy" has over Notion at least.

I was a bit sad to see the roadmap, with the mention of a "plugin market." If the project is going to be open source, it should be open source everywhere, not stuffed with some micro-transactions like a crappy EA game.

Anyways, I do hope to see this project turn into something good. And the comments above are not meant to negative per say, but more of food for thought, suggestions, or what not. However I do see that "flowy" has a long way to go before its anything like Notion. Personally I don't use Notion, as I'm so over all these "team" based collaboration apps/services/websites popping up. I prefer the stone-age where my files are mine, and I don't need to signup for some stupid account to use something. I just keep everything in .txt files 😝

**Comment by annieappflowy** [comment]
Hi @mB-PiBox ,

Thanks a lot for taking time to provide your feedback. We will consider your suggestions on the design once we have a designer in place. At the moment, we are just a small team of two. Given that it takes tremendous resource to build a reliable collaborative software, I share your opinion that we have a long way to go, not to mention we started AppFlowy a few months ago by a single developer. To the transparency part, I appreciate your opinion, and we will bring more transparency into the roadmap in the near term.

Annie

**Reply by mcbaer** [reply]
> "collaborative"

This is what you should be moving away from.. there are so many "collaborative" document apps/services out there already.

I deal with so many different clients in my day to day, having to keep track of all the different services each one uses is crazy, to say the least. Its like every other client is using something different, from MS Teams/Office, to Quip, to Notion, to Google Docs, to Zoho, and the list goes on.
